### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by jarrettj** [aws/awsmobile-cli#185]
Hi,

Good day.

"Go to the AWS Mobile Docs to learn more about the usage of the awsmobile-cli and some sample codes to get your app up and running."

That links you to amplify cli? Where have the docs gone?

Regards.

**Comment by Sigmus** [comment]
I'm starting to notice some redirects in the docs as I follow them. Is ```awsmobile-cli``` being decommissioned and replaced by ```amplify```?

**Comment by jarrettj** [comment]
I have no idea, but not a cool way to do things. How do we migrate an existing app. I'm setting up all the same stuff again! :(

**Comment by ryanwmarsh** [comment]
I lost about a day on this as well. If Amazon is in a transition period from awsmobile-cli to amplify that's fine just please **make it clear**. I created several amplify projects and couldn't figure out why they weren't showing up in MobileHub like awsmobile-cli projects used to do. Very frustrating. re:Invent is about three months away perhaps they're waiting for some big announcement that Amplify is replacing MobileHub or something.

**Comment by jarrettj** [comment]
I fully agree man, just some communication would be great. It’s very frustrating. Trying to use this for a production application but it feels like the tools are in alpha!

**Comment by swaminator** [comment]
@jarrettj @ryanwmarsh sorry for the confusion! Please check out our latest announcement: https://aws.amazon.com/blogs/mobile/announcing-the-aws-amplify-cli-toolchain/
The relevant bits: "Existing Mobile Hub projects continue to work without requiring any app changes. If you’re using the AWS Mobile CLI for existing projects, you can also continue to use that older CLI. However, going forward, new features will be added to the AWS Amplify CLI toolchain which does not use Mobile Hub." 

You can continue to use the AWS Mobile CLI for your existing projects (and it'll work with mobile hub in the same way), but for new projects we recommend starting here: https://aws-amplify.github.io/media/get_started

**Comment by ryanwmarsh** [comment]
I think this is safe to close re: @swaminator's comment

**Comment by 0xR** [comment]
I'm missing a migration path to amplify-cli

**Comment by bytheway875** [comment]
Definitely frustrating to have to go and learn something new when every AWS product takes such an investment to learn to proficiency. Not sure if it makes sense to choose Amazon cli tools for things like this if ya'll are going to up and reinvent the wheel every year or so... would be nice if you guys could invest in making ONE production-ready, fully-featured tool instead of multiple almost-there solutions to the same problems.
